src/os_auth/local-server.c

```diff
@@ -240,18 +242,27 @@ char* local_dispatch(const char *input) {
             key_hash = (item = cJSON_GetObjectItem(arguments, "key_hash"), item) ? item->valuestring : NULL;
             key = (item = cJSON_GetObjectItem(arguments, "key"), item) ? item->valuestring : NULL;
 
-            if (item = cJSON_GetObjectItem(arguments, "force"), item) {
-                if (item->valueint == -1) {
-                    force_options.enabled = false;
-                } else {
-                    force_options.enabled = true;
-                    force_options.connection_time = item->valueint;
+            if (force = cJSON_GetObjectItem(arguments, "force"), force) {
+                if (item = cJSON_GetObjectItem(force, "enabled"), item) {
+                    force_options.enabled = (bool)item->valueint;
+                }
+                if (item = cJSON_GetObjectItem(force, "key_mismatch"), item) {
```

Comment:
I would take all the next items only if `force_options.enabled` is `true`. If not, I think it is not necessary.

Reply:
We agreed with the team that we will keep the whole parsing independently on whether or not the forcing is enabled or disabled.